src/Url.php

```diff
@@ -39,67 +40,85 @@ class Url
     private $priority;
 
     /**
-     * @return mixed
-     */
-    public function getLocation()
-    {
-        return $this->location;
-    }
-
-    /**
      * Url constructor.
-     * @param string $location location item URL
+     * @param string $location URL of the page.
+     * @param string $lastModified the date of last modification of the file
+     * @param string $changeFrequency how frequently the page is likely to change
+     * @param string|float $priority the priority of this URL relative to other
+     * URLs on site
      * @throws \InvalidArgumentException
      */
-    public function __construct($location)
+    public function __construct($location, $lastModified = null, $changeFrequency = null, $priority = null)
```

Comment:
It's true, but then need create UrlInterface because Google video sitemap use other fields\tags.